generator/src/routes/styles.css

```diff
@@ -1,5 +1,9 @@
 @import "@fontsource/fira-mono";
 
+* {
+  box-sizing: border-box;
+  font-family: "Poppins", sans-serif;
+}
 :root {
   --font-body: Arial, -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, Oxygen, Ubuntu,
```

Comment:
This should be removed

Reply:
box-sizing: border-box;
This should be used for all html elements.

Reply:
The comment is for the `--font-body` -line, GitHub doesn't apparently want to show that clearly